integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -87,10 +87,10 @@ def save_input_values(inputs: Sequence[np.ndarray],
 
 
 def _setup_mlir_crash_reproducer(
-    function: Callable[[Any], Any],
+    function: Any,  # pytype doesn't support arbitrary Callable[*args, **kwargs]
```

Comment:
I found a thread about this somewhere. I think it said there is a way to write this. Maybe worth investigating, but can be a followup

Reply:
SGTM. Could you send a link?

Reply:
https://github.com/python/mypy/issues/5876. I would have to understand pytype better to understand all that's going on in that thread. https://github.com/python/typing/issues/264#issuecomment-436525286 and https://stackoverflow.com/q/57837609 may also help

Reply:
Yeah, I was looking through those. The first one would be ideal if it were implemented. But the other two make assumptions about types that the function can ingest, which would be inappropriate in our case.